### PR TITLE
Show durations in the UI in useful units (#200)

### LINKED ISSUE #200: Show duration in the UI in useful units
`2,883,499 events · 2 workers · 28808404.3ms`

In this (and many other places) in the viewer, show the durations in the UI in a "sensible human unit" (in this case 8 hours).

### PR DESCRIPTION
**Pull request by rcoh**
Fixes #200.

Previously the viewer header rendered trace durations in raw milliseconds, so a long trace showed up as e.g. `28808404.3ms` (~8 hours). Pick a sensible unit based on magnitude (ns/µs/ms/s/m/h/d) so the header reads as `8h 0m 8s` instead.

Changes:
- New `ui/format.js` with `formatHumanDuration(ns)`
- Viewer header uses it in place of `${durMs.toFixed(1)}ms`
- Unit tests in `ui/test_format.js` (runnable with `node`)

Scope kept intentionally small: only the top-level `tb-stats` header is changed. Other places (poll durations, park durations, task lifetimes) already switch between µs and ms and don't hit the > 1 second range in practice.

### CODE REVIEW

File: dial9-viewer/ui/test_format.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+assertEq(formatHumanDuration(3_599_000_000_000), "59m 59.0s", "just under 1 hour");
+
+// Hours (>= 60 minutes)
+assertEq(formatHumanDuration(3_600_000_000_000), "1h 0m 0s", "1 hour");
+// The bug report case: 28,808,404.3 ms ≈ 8h 0m 8s
+assertEq(formatHumanDuration(28_808_404_300_000), "8h 0m 8s", "8-hour trace from issue #200");
+
+// Days (>= 24 hours)
+assertEq(formatHumanDuration(86_400_000_000_000), "1d 0h 0m", "1 day");
+assertEq(formatHumanDuration(90_000_000_000_000), "1d 1h 0m", "1d 1h");
```
Comment (jlizen) [inline dial9-viewer/ui/test_format.js:56]: probably throw in a few edge case tests (Nan, infinity, -1, undefined), though the handling looks fine to me

File: dial9-viewer/ui/test_format.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+// Sub-microsecond → ns
+assertEq(formatHumanDuration(0), "0ns", "zero");
+assertEq(formatHumanDuration(500), "500ns", "500 ns");
+assertEq(formatHumanDuration(999), "999ns", "999 ns");
+
+// Microseconds
+assertEq(formatHumanDuration(1_000), "1.0µs", "1 µs");
+assertEq(formatHumanDuration(1_500), "1.5µs", "1.5 µs");
+assertEq(formatHumanDuration(999_999), "1000.0µs", "just under 1 ms");
```
Comment (jlizen) [inline dial9-viewer/ui/test_format.js:33]: This behavior is a bit odd, as a viewer I might expect to see this show as 1ms given how it rounds. Might consider setting the threshold to be a bit lower (ie, whether it would round to 999 or less)

**Review by jlizen — APPROVED** [review]
